### PR TITLE
add powershell build script

### PR DESCRIPTION
**Pull request by y-ack**

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+function nocache($name) {
+	return "$name`?$(unixtime (Get-Item $name).LastWriteTime)"
+}
+
+Write-Host "creating _build.html"
+$f = (Get-Content ./index.html -Raw)
+$start = $f.IndexOf("<!--START-->")
+$end = $f.IndexOf("<!--END-->") + "<!--END-->".Length
+$f.Remove($start,$end-$start).Insert($start,"<link rel=`"stylesheet`" href=`"$(nocache 'resource/_build.css')`">`n<script src=`"$(nocache 'resource/_build.js')`">") | sc _build.html
```
Comment (12Me21) [inline build.ps1:38]: missing </script> tag

File: build.ps1 [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+function isnewer([System.IO.FileInfo] $file, $destfilename = $null) {
+    if ($destfilename -eq $null) {$destfilename = $file.Name}
+	$destfile = [System.IO.FileInfo]::new((Join-Path $dest $destfilename))
+	if ($destfile.Exists) {
+		return ((Compare-Object $file $destfile -IncludeEqual -Property LastWriteTime)[0].SideIndicator -eq "<=")
```
Comment (12Me21) [inline build.ps1:44]: should be `=>`, I think

Comment (y-ack) [reply build.ps1:44]: yeah.

File: build.ps1 (hunk 1 above)
Comment (y-ack) [reply build.ps1]: should be fixed by https://github.com/12Me21/sbs2/pull/5/commits/70fd33655640e380842329e63581ecb5652d09cb

**Pull request merged**